variables.tf

```diff
@@ -190,3 +190,9 @@ variable "tags" {
   type        = map(string)
   default     = {}
 }
+
+variable "vpc_cidr_block" {
+  description = "VPC CIDR block"
+  type        = string
+  default     = "172.16.0.0/20"
```

Comment:
The previous default was `10.0.0.0/16` since we now have a new default value of `172.16.0.0/20` this will cause anyone who upgrades that doesn't change set this value to the previous one to cause a bunch of recreates. 

Could the old value `10.0.0.0/16` stay as the default? Or is there a reason we would want to change this. 

TLDR: If we default to the old value then the VPC's won't need to be replaced.